src/dhtproto/client/legacy/internal/helper/RetryHandshake.d

```diff
@@ -203,6 +182,8 @@ class RetryHandshake
             {
                 this.established_nodes[node_hash] = true;
                 this.one_node_handshake_dg(info.nodeitem);
+                this.log.info("Handshake succeeded on {}:{}.",
```

Comment:
This is going to be very noisy in production uses.  Are you sure it should be `info` and not `trace` ... ?

Reply:
Hmm good question. I figured that as this happens only once ever per node, it's not too bad. And it's pretty useful logging.

Reply:
Yup, on balance I agree.  It's straightforward enough to redirect the logging for this module to another file (this is useful with e.g. the new `Mirror` class, where the entries are not just once on app startup).

Reply:
That's good. (I'd wondered about that, as the logger name is determined at run-time.)